src/main/resources/config/liquibase/changelog/20200117213324_changelog.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <changeSet id="20200117213324-1" author="ungar">
+        <addColumn tableName="submission">
+            <column name="build_artifact" type="Boolean" defaultValueBoolean="false">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="20200117213324-2" author="ungar">
+        <sql>
+            update submission
+                set build_artifact = true
+                where discriminator = 'P'
+                and id in (
+                    select submission_id
+                    from result
+                    where build_artifact
+                )
+        </sql>
+    </changeSet>
+    <changeSet id="20200117213324-3" author="ungar">
+        <dropColumn tableName="result" columnName="build_artifact" />
```

Comment:
remove and execute this changelog some days later for easier migration

Reply:
Removed the dropColumn change